Source/App/HeadlessMain.cpp

```
@@ -253,6 +255,66 @@ static AudioGraph buildLoopGraph(const std::shared_ptr<AudioBuffer>& source,
+static std::string makeRoundTripPath(const std::string& name) {
+    namespace fs = std::filesystem;
+    std::error_code ec;
+    fs::path base = fs::temp_directory_path() / "AestraHeadless";
+    fs::create_directories(base, ec);
+    const auto stamp = std::chrono::steady_clock::now().time_since_epoch().count();
+    std::string safeName = name.empty() ? "roundtrip" : name;
+    fs::path file = base / (safeName + "_" + std::to_string(stamp) + ".aes");
+    return file.string();
+}
+
+static RunMetrics renderEngine(AudioEngine& engine,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)